tests/page-objects/reports/reports.po.js

```diff
@@ -82,30 +91,70 @@ module.exports = {
     expect(element.all(by.css('#reports-list li[data-record-id="' + savedUuids[1] + '"]')).count()).toBe(1);
   },
 
+  deleteSelectedReportsNative: async (savedUuids) => {
+    await element(by.css('.action-container .detail-actions .delete-all')).click();
+    const confirmButton = element(by.css('.btn.submit.btn-danger'));
+    await helper.waitElementToBeClickable(confirmButton, 5000);
+    await confirmButton.click();
+    await helper.waitElementToBeClickable(confirmButton, 5000);
+    await confirmButton.click();
+    await helper.waitElementToPresent(element(by.css('#reports-list li')), 30000);
```

Comment:
Yikes, are there really cases where this can take 30 seconds!?

Reply:
I didn't even see that was 30 seconds. I'm going to remove that and let the default 10 seconds.